providers/base/units/dock/jobs.pxu

```diff
@@ -2355,85 +2355,535 @@ _description: Check dock MAC address pass-through function, please make sure fol
+    8. Connect to a USB3.0 storage.
+    9. Connect to a headset.
 _verification:
-     1. Did the display works fine?
-     2. Did the storage works fine?
-     3. Did the audio works fine?
-     4. Did the network works fine?
+    Mark it Pass if you have connected all I/O ports as mentioned above.
 
 id: dock/all-reboot
 category_id: dock
 plugin: manual
 estimated_duration: 300.00
 _purpose:
-     This test is to test devices still working after reboot
+    This test is to check if the device is still working after reboot.
 _steps:
-     Reboot the system.
+    Reboot the system.
 _verification:
-     1. Did the display works fine after reboot?
-     2. Did the storage works fine after reboot?
-     3. Did the audio works fine after reboot?
-     4. Did the network works fine after reboot?
+    Mark it Pass if you have rebooted the system.
 
 id: dock/all-hotplug
 category_id: dock
 plugin: manual
 estimated_duration: 300.00
 _purpose:
-     This test is to test devices still working after hotplug
+    This test is to check if the device is still working after hotplug.
 _steps:
-     1. Unplug docking station.
-     2. Waiting 30 seconds.
-     3. Plug in docking station.
+    1. Unplug the docking station.
+    2. Wait for 30 seconds.
+    3. Plug in the docking station.
 _verification:
-     1. Did the display works fine after hotplug?
-     2. Did the storage works fine after hotplug?
-     3. Did the audio works fine after hotplug?
-     4. Did the network works fine after hotplug?
+    Mark it Pass if you have hotpluged the docking station.
 
 id: dock/all-suspend
 category_id: dock
 plugin: manual
 estimated_duration: 300.00
 _purpose:
-     This test is to test devices still working after suspend
+    This test is to check if the device is still working after suspend.
 _steps:
-     1. Suspend the system.
-     2. Waiting 30 seconds.
-     3. Resume the system.
+    1. Suspend the system.
+    2. Waiting 30 seconds.
+    3. Resume the system.
 _verification:
-     1. Did the display works fine after suspend?
-     2. Did the storage works fine after suspend?
-     3. Did the audio works fine after suspend?
-     4. Did the network works fine after suspend?
+    Mark it Pass if you have suspended and resumed the system.
 
 id: dock/all-poweroff
 category_id: dock
 plugin: manual
 estimated_duration: 300.00
 _purpose:
-     This test is to test devices still working after poweroff
+    This test is to check if the device is still working after power-off.
 _steps:
-     1. Poweroff the system.
-     2. Waiting 30 seconds.
-     3. Poweron the system.
+    1. Poweroff the system.
+    2. Wait for 30 seconds.
+    3. Poweron the system.
 _verification:
-     1. Did the display works fine after poweroff?
-     2. Did the storage works fine after poweroff?
-     3. Did the audio works fine after poweroff?
-     4. Did the network works fine after poweroff?
+    Mark it Pass if you have powered on and powered off the system.
```

Comment:
Wouldn't it be the contrary? First power it off, then power it back on successfully?

Reply:
Yep, you are right, typo.

Reply:
Done.